src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,11 +8,22 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
-    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
-    public static final String ADDRESS_VALIDATION_REGEX = ".+";
+    public static final String EXAMPLE = "123, Bishan Ave 5, #15-12, 460123";
+    public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in BLOCK, STREET, UNIT, POSTAL_CODE format";
+    public static final String ADDRESS_VALIDATION_REGEX = ".+,.+,.+,.+";
+    public static final String SPLIT_TOKEN = ", ";
+
+    public final int BLOCK_INDEX = 0;
+    public final int STREET_INDEX = 1;
+    public final int UNIT_INDEX = 2;
+    public final int POSTAL_CODE_INDEX = 3;
 
     public final String value;
+    public final Block block;
```

Comment:
Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.